### PR TITLE
Tidy up for Voiceover.

### PR DESCRIPTION
**Pull request by pixlwave**
Small PR with
- A couple of tweaks for voiceover (its not comprehensive but its more usable)
- Fix for the bug report header
- Removed redundant strings there were added recently.

### CODE REVIEW

File: ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemBubbledStylerView.swift [hunk 1]
```diff
@@ -98,14 +101,13 @@ struct TimelineItemBubbledStylerView<Content: View>: View {
 
     @ViewBuilder
     var styledContentOutgoing: some View {
-        if timelineItem.inGroupState == .single || timelineItem.inGroupState == .beginning {
-            Spacer()
-                .frame(height: 8)
-        }
+        let topPadding: CGFloat? = timelineItem.inGroupState == .single || timelineItem.inGroupState == .beginning ? 8 : 0
```
Comment (pixlwave) [inline ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemBubbledStylerView.swift:104]: That `Spacer` wasn't in the VStack, so we returned 2 view here and for some reason the spacer was added to the VO hierarchy 🤨 Seems simpler to fix this with padding

File: ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemBubbledStylerView.swift [hunk 2]
```diff
@@ -171,7 +174,7 @@ struct TimelineItemBubbledStylerView_Previews: PreviewProvider {
 
     @ViewBuilder
     static var body: some View {
-        VStack(alignment: .leading) {
+        VStack(alignment: .leading, spacing: 2) {
```
Comment (pixlwave) [inline ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemBubbledStylerView.swift:177]: Aligns with the spacing used by `List`.

File: ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemBubbledStylerView.swift [hunk 3]
```diff
@@ -60,12 +60,14 @@ struct TimelineItemBubbledStylerView<Content: View>: View {
...
                 HStack(alignment: .top, spacing: 4) {
                     TimelineSenderAvatarView(timelineItem: timelineItem)
+                        .accessibilityHidden(true)
                     Text(timelineItem.senderDisplayName ?? timelineItem.senderId)
                         .font(.element.footnoteBold)
                         .foregroundColor(.element.primaryContent)
                         .lineLimit(1)
                         .padding(.vertical, senderNameVerticalPadding)
                 }
+                .accessibilityElement(children: .combine)
```
Comment (stefanceriu) [inline ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemBubbledStylerView.swift:70]: Nice!

**Review by stefanceriu — APPROVED** [review]
Looks great! 👍

**Pull request merged**